waiter/src/waiter/work_stealing.clj

```diff
@@ -123,33 +128,35 @@
                          ([_]
                            (let [router-id->metrics (service-id->router-id->metrics service-id)
                                  _ (log/trace label "received metrics from" (count router-id->metrics) "routers")
-                                 slots-offered (count request-id->work-stealer)
                                  extra-slots (-> (router-id->metrics router-id)
-                                                 (assoc "slots-offered" slots-offered)
```

Comment:
So, previously, we were saying that more slots a router had already loaned out, the more slots the router could now loan out?

Reply:
:(